cluster-operator/src/main/java/io/strimzi/operator/cluster/model/EntityTopicOperator.java

```diff
@@ -283,6 +291,11 @@ public static EntityTopicOperator fromCrd(Reconciliation reconciliation, Kafka k
         varList.add(buildEnvVar(ENV_VAR_SECURITY_PROTOCOL, EntityTopicOperatorSpec.DEFAULT_SECURITY_PROTOCOL));
         varList.add(buildEnvVar(ENV_VAR_TLS_ENABLED, Boolean.toString(true)));
         varList.add(buildEnvVar(ENV_VAR_STRIMZI_GC_LOG_ENABLED, String.valueOf(gcLoggingEnabled)));
+
+        varList.add(buildEnvVarFromSecret(ENV_VAR_EO_CERTIFICATE_CHAIN_DATA, EntityOperator.secretName(cluster), "entity-operator.crt"));
+        varList.add(buildEnvVarFromSecret(ENV_VAR_EO_KEY_SECRET_DATA, EntityOperator.secretName(cluster), "entity-operator.key"));
+        varList.add(buildEnvVar(ENV_VAR_CLUSTER_CA_CERT_SECRET_DATA, Util.certsToPemString(clusterCaCertSecret)));
```

Comment:
This should be mounted from secret.

Reply:
Yeah, I know. This is a workaround as this secret can contain multiple `*.crt` properties. The `buildEnvVarFromSecret` method allows us to load just one key, so we would need envar per key(?). I did not find any better way how to implement this :(

Reply:
Well, I think you need to make sure we have one file with all somewhere first then I guess. This is not just a TO problem. That seems to be a general problem for all clients.

Reply:
What's preventing concatenating the PEM encoded certs?